.github/workflows/regression.yml

```diff
@@ -0,0 +1,60 @@
+name: regression
+
+# start job only for PRs when a label is added.
+on: push
+  # pull_request:
+  #   types: [labeled]
+
+jobs:
+  regresstion:
+    # if: contains(github.event.pull_request.labels.*.name, 'run-regression')
```

Comment:
Maybe given the fact that uses a runner under your account it would be better to run it only with the labels to avoid hammering the runner on every commit?

Reply:
Reverted back in [04e91d6](https://github.com/NNPDF/pineko/pull/215/commits/04e91d6a2abc0577ab8c5c6310b417f0e3a2676f), now given that there are more datasets.